sopel/modules/bugzilla.py

```diff
@@ -42,34 +39,18 @@ def configure(config):
 
 
 def setup(bot):
-    global regex
     bot.config.define_section('bugzilla', BugzillaSection)
 
-    if not bot.config.bugzilla.domains:
-        return
-
-    domains = '|'.join(bot.config.bugzilla.domains)
-    regex = re.compile((r'https?://(%s)'
-                        r'(/show_bug.cgi\?\S*?)'
-                        r'(id=\d+)')
-                       % domains)
-    bot.register_url_callback(regex, show_bug)
-
-
-def shutdown(bot):
-    bot.unregister_url_callback(regex, show_bug)
-
 
-@rule(r'.*https?://(\S+?)'
-      r'(/show_bug.cgi\?\S*?)'
-      r'(id=\d+).*')
+@url(r'https?://(\S+?)'
+     r'(/show_bug.cgi\?\S*?)'
+     r'(id=\d+).*')
```

Comment:
I thought plugins like this were the motivator for creating `plugin.url_lazy`, but you didn't switch to it here. Why not?

Reply:
Draft, mistake. :stuck_out_tongue:

Reply:
Fixed, with a bonus.